src/reorders/reorderCatArgs.ts

```diff
@@ -17,6 +17,8 @@ export default (function reorderCatArgs(
       catArgsManager.makeNewCatArrs(store, inst_axis, cat_index);
       // update the attribute
       catArgsManager.updateCatArgsAttribute(inst_axis, cat_index);
+
+      catArgsManager.regenerateCatArgsArrs(store); // added to preserve the current state fixed
     }
```

Comment:
For better readability, can we stick to comments on the top? The line above this has the comment above it, and the comment here is to the side. Makes it hard to read.